website/docs/about/contribution.md

```diff
@@ -1,34 +1,200 @@
+- What are the open questions, risks or unknowns for this project
+- What are the stakeholders that need to be involved or whose approval is required
+- What are the first actionable steps and milestones (if they are already known)
+- How the team plans to communicate about the project
+
+### Execution
+
+The designers and/or developers from the HDS and consumer teams will work together, according to the scope and timeline defined first in the kickoff, and then reviewed during the implementation (not unusual, due to gained knowledge of the problem space).
+It’s not possible to describe here and/or generalize what this phase will look like, because it depends on too many factors. You can have an idea of what kind of work can be executed by looking at some of the examples of previous collaborations under the [Successful collaborations section](#examples-of-successful-collaborations). You can also look at [the process followed by the HDS team when building a net new component](https://github.com/hashicorp/design-system/blob/main/packages/components/NEW-COMPONENT-CHECKLIST.md) as reference of the typical work that is involved.
+
+### Conclusion
+
+At the end of the process:
+- The HDS team will (if applicable):
+    - Release the artifacts created/ported/updated in collaboration with the consumer team
+    - Document these artifacts in Figma and/or on the Helios website
+    - Publicize the release of these new artifacts, as well as the fact that they were the result of of a joint collaboration with the consumer team
+- The consumer team will:
+    - Follow the guidance provided by the HDS team in the follow-up work related to the collaboration effort
+    - Have a clear plan to adopt the artifacts produced during the collaboration and published as official HDS resources (foundations, components, etc)
+
+---
+
+## Examples of successful collaborations
+
+We list below some examples of successfull collaborations with our consumers, happened in the last year or so. They are not only “success stories” but also as examples of “what went well”, to be used as a reference of what makes a collaboration successful for both parties.
+
+### Dark mode for HashiCorp Developer
+
+The marketing team was given the goal to launch a “dark mode” version of the [HashiCorp Developer website](https://developer.hashicorp.com/). This task necessarily required the involvement of the HDS team, since there was no official support of “modes” in the design system's foundational color palette. After an initial evaluation of the effort required to introduce such support in the current system, and considered the short timeframe, we decided to take a slightly different approach: instead of the Developer team having to wait for the HDS team to come up with a color palette for the “dark” mode, the two teams would worked together on the problem, in a timeboxed spike, with the goal to create a “dark” mode palette **for the Developer website**, not for the design system.
```

Comment:
```suggestion
The marketing team was given the goal of launching a “dark mode” version of the [HashiCorp Developer website](https://developer.hashicorp.com/). This task necessarily required the involvement of the HDS team since there was no official support for “modes” in the design system’s foundational color palette. After an initial evaluation of the effort required to introduce such support in the current system and considering the short timeframe, we decided to take a slightly different approach: instead of the Developer team having to wait for the HDS team to come up with a color palette for the “dark” mode, the two teams would work together on the problem in a timeboxed spike, with the goal to create a “dark” mode palette **for the Developer website**, not for the design system.
```